src/test/suite/editors/activeConnectionCodeLensProvider.test.ts

```diff
@@ -21,34 +21,38 @@ suite('Active Connection CodeLens Provider Test Suite', () => {
     extensionContextStub
   );
   const testStatusView = new StatusView(extensionContextStub);
+  let testConnectionController: ConnectionController;
+  let testCodeLensProvider: ActiveConnectionCodeLensProvider;
+  let sandbox: sinon.SinonSandbox;
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
```

Comment:
I think we can create one sandbox per suite instead of putting those inside beforeEach: https://sinonjs.org/releases/latest/sandbox/